### PR TITLE
[POT] Update patterns with transpose

### PR DESCRIPTION
**Pull request by IndiraSalyahova**

### CODE REVIEW

File: tools/pot/openvino/tools/pot/graph/special_patterns.py [hunk 1]
```diff
@@ -256,3 +256,52 @@ def create_equal_logicalnot_pattern():
     pattern.append_single_op('Equal', 'equal')
     pattern.append_single_op('LogicalNot', 'logicalnot')
     return pattern.set_name('equal_logicalnot').pattern
+
+
+@registry_ignore_patterns('inputs')
```
Comment (AlexKoff88) [inline tools/pot/openvino/tools/pot/graph/special_patterns.py:261]: @IndiraSalyahova, @KodiaqQ, any chance we can parametrize the presence of Concat in the existing input-related patterns? or do we need a separate set of patterns as it is done in this PR?
If the latter is right, I would recommend moving and placing all these patterns right after the first set of input-related patterns. The same is applicable for documentation.

Comment (KodiaqQ) [reply tools/pot/openvino/tools/pot/graph/special_patterns.py:261]: In the current implementation, we will need to create a separate set of input patterns with the Concat. We can iterate over different types of layers when creating a pattern, but on the condition that the layer is exactly in the pattern. Here is the case when the layer may or may not be.

Comment (KodiaqQ) [reply tools/pot/openvino/tools/pot/graph/special_patterns.py:261]: Do we need to create the model examples with the corresponding patterns for our graph tests? Or are they already presented, @IndiraSalyahova?

Comment (AlexKoff88) [reply tools/pot/openvino/tools/pot/graph/special_patterns.py:261]: Let's validate at least the basic ones where we have intput->transpose->ScaleShift

Comment (AlexKoff88) [reply tools/pot/openvino/tools/pot/graph/special_patterns.py:261]: @IndiraSalyahova, can you please introduce the test and fix CI we need it asap.

File: tools/pot/tests/test_graph.py [hunk 2]
```diff
@@ -211,7 +211,10 @@ def test_cutting_fq_layers(_params, tmp_path, models):
     ('scaleshift_fuse', 'dldt'),
     ('scaleshift_no_fuse_1', 'dldt'),
     ('scaleshift_no_fuse_2', 'dldt'),
-    ('matmul_divide_const', 'dldt')
+    ('matmul_divide_const', 'dldt'),
+    ('inception-resnet-v2', 'tf'),
```
Comment (AlexKoff88) [inline tools/pot/tests/test_graph.py:215]: @IndiraSalyahova, is this XML was taken as is? We cannot redistribute models even from OMZ.

**Review by KodiaqQ — APPROVED** [review]

**Pull request merged**